components/top-area/TopArea.js

```diff
@@ -102,8 +102,7 @@ const TopArea = (props) => {
                           </div>
                         </div>
                         <p className="sm">
-                          Build a basic Todo application and learn about
-                          Authentication and Gaia storage.
+                          Build a DApp to learn about the Blockstack platform and how to join Application Mining.
```

Comment:
```suggestion
                           Build a DApp to learn about the Blockstack platform and how to join App Mining.
```

I think we typically refer to it as `App Mining` everywhere, not `application mining` :)

Reply:
gah!

Reply:
fixed